apps/docs/src/app/platform/component-docs/platform-menu-button/platform-menu-button-examples/platform-menu-button-compact-examples.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'fdp-platform-menu-button-compact-example',
+    templateUrl: './platform-menu-button-compact-example.component.html',
+})
+export class PlatformMenuButtonCompactExampleComponent {
+    basicMenuData: any[] = [];
```

Comment:
Decide if you use explicite typing and use one convention

Reply:
resolved